src/site/content/en/payments/handling-optional-payment-information-with-a-service-worker/index.md

```diff
@@ -0,0 +1,465 @@
+---
+layout: post
+title: Handling optional payment information with a service worker
+subhead: |
+  How to adapt your web-based payment app to Web Payments and provide a better user experience for customers.
+authors:
+  - agektmr
+date: 2020-08-14
+description: |
+  Once a web-based payment app is registered, it's ready to accept payment requests from merchants. This article teaches you how to orchestrate a payment transaction from a service worker during runtime.
+tags:
+  - payments
```

Comment:
```suggestion
  - blog
  - payments
  - service-worker
  - monetization
```
Let's increase the visibility of this new content by cross-posting to the blog? Adding this tag just causes the content to show up in the blog feed. This will also ensure that it gets posted to HN (because someone has created a bot that reads our RSS feed and creates a thread for every new page of blog content).

If you want the posts to show up on the homepage too (under "Latest Stories") you'll need to add hero images

Reply:
TIL about `blog` tag causes it to show up under `/blog`. This is a great tip. However, if I knew this I could have done this from the beginning of the series. I feel it's odd to add this suddenly to `/blog` without context. What happens if I add `blog` to the existing articles?

Reply:
FYI I think I started following the policy of adding `blog` to everything after you posted the first wave of content. 

If you add `blog` to the existing articles they'll start showing up on web.dev/blog, which SGTM. There may be a chance that the bots that watch our feed (we don't control them) will detect all of the posts and post them all to HN at the same time. Or they won't detect them at all. So maybe try adding it to one of the existing posts first, and we'll see what the bot does?

Reply:
OK, I have added `blog` to all payment articles.